CMakeLists.txt

```diff
@@ -1018,21 +1018,19 @@ configure_file(
 if(CONFIG_CPU_HAS_MPU AND CONFIG_USERSPACE)
 
     if(CONFIG_MPU_REQUIRES_POWER_OF_TWO_ALIGNMENT AND CONFIG_APP_SHARED_MEM)
-        set(GEN_APP_SMEM $ENV{ZEPHYR_BASE}/scripts/gen_app_smem.py)
+        set(GEN_APP_SMEM $ENV{ZEPHYR_BASE}/scripts/gen_app_partitions.py)
```

Comment:
Might want to consider changing "GEN_APP_SMEM".

Reply:
Or just eliminate it.  All we do is set this variable in one place so we can expand it somewhere else.  I don't think it's adding any clarity or useful abstraction at all vs. just naming the script you are going to run at the place you are going to run it.